2023/10/28 13:08:47 failed to dial tcp connection, retrying
[2023-10-28T13:08:48.941Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-10-28T13:08:48.942Z] info  Using user-data-dir ~/.local/share/code-server
====================================>token: bdc29b37-156f-4585-b597-2bed20043f4c
[2023-10-28T13:08:48.958Z] info  Using config file ~/.config/code-server/config.yaml
[2023-10-28T13:08:48.959Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-10-28T13:08:48.959Z] info    - Authentication is disabled
[2023-10-28T13:08:48.959Z] info    - Not serving HTTPS
[13:08:51] 




[13:08:51] Extension host agent started.
[13:08:53] [127.0.0.1][2be62221][ManagementConnection] New connection established.
[13:08:53] [127.0.0.1][81e88499][ExtensionHostConnection] New connection established.
[13:08:53] [127.0.0.1][81e88499][ExtensionHostConnection] <378> Launched Extension Host Process.
[13:08:57] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[13:09:07] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-10-28T13:11:03.609Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8986f1]

goroutine 356 [running]:
http_reverse_proxy/client.sendHttpRequestAndReplyHttpResponse(0x0?, 0x0?, 0xc000330040, 0xc00011c060)
	/builds/hitminer/http_reverse_proxy/client/http_client.go:51 +0xb1
created by http_reverse_proxy/client.startHttpClient
	/builds/hitminer/http_reverse_proxy/client/http_client.go:34 +0x1ea
